app/views/dates/difference.php

```diff
@@ -5,7 +5,7 @@
     <a href="<?= URLROOT; ?>/dates">Back</a>
     <div class="card card-body bg-light mt-5">
         <h3><strong><?= $data[0]; ?></strong>
-            <?php if($data['returnformat'] == 'days' || $data['returnformat'] == 'weeks' || $data['returnformat'] == 'years') : ?>
+            <?php if($data['returnformat'] == 'days' || $data['returnformat'] == 'weeks' || $data['returnformat'] == 'years' || $data['returnformat'] == 'weekdays') : ?>
```

Comment:
This might be neater:
`if (in_array($data['returnformat'], ['days', 'weeks', 'years', 'weekdays']))`

Reply:
ooo nice